iib/workers/config.py

```diff
@@ -50,6 +53,7 @@ class DevelopmentConfig(Config):
 class TestingConfig(DevelopmentConfig):
     """The testing IIB Celery configuration."""
 
+    iib_greenwave_url = 'some_url'
```

Comment:
I think this is not an issue as long as the configuration options are documented.

Reply:
Can I please ask if we agree on one of the suggestion here? It would conflict with Matt's suggestion of not having Gating compulsory for all IIB deployments. Thoughts?

fwiw, I agree with Matt's comment here :)

Reply:
Documentation is great, but code is better :) 
I'm happy to drop this comment though.
As I mentioned, it's an existing pattern.